src/logging/logniWrapper.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import logni from 'logni.js'
+
+/**
+ * Simple wrapper of logni.
+ */
+const logniWrapper = {
```

Comment:
Ten wrapper takto nedava smysl - dela to same co logni. V kosiku wrapper vzdy pridaval id kosiku - jako jednoznacny identifikator. Mamem nejaky takovy identifikator k dispozici tady? Nejaky token nebo tak neco?